source/camel-cased-properties-deep.d.ts

```diff
@@ -40,8 +41,9 @@ const result: CamelCasedPropertiesDeep<UserWithFriends> = {
 
 @category Template Literals
 */
-export type CamelCasedPropertiesDeep<Value> = Value extends Function
-	? Value
+export type CamelCasedPropertiesDeep<Value, Exclude = never>
+  = Value extends Exclude | StandartBuildInTypes
```

Comment:
Skip adding the `StandartBuildInTypes` for now, when/if it becomes evident that we need, then we can extract it

Reply:
OK, remove it at all or just remove its union with `Exclude`?

Reply:
Should I return `extends Function` to support backward combability?